### ISSUE TITLE
atom-text-editor.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by ghuroo** [willyelm/atom-bugs#13]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
atom-text-editor.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor-element.js:68:18)
atom.workspace.observeTextEditors (/Users/danielneves/.atom/packages/atom-bugs/lib/bugs-view.js:182:29)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:334:16)
Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:265:34)
```

**Comment by willyelm** [comment]
Fixed on atom-bugs@v0.21.0

**Comment by ghuroo** [comment]
@willyelm cheers! thank you
